compiler/scopelang/ast.ml

```diff
@@ -199,4 +328,19 @@ let make_let_in
        (Pos.get_position (Bindlib.unbox e2)))
     (Bindlib.box_list [e1])
 
+let make_default ?(pos = Pos.no_pos) exceptions just cons =
```

Comment:
Ok so you integrate some optimization in there already. Might be worth mentioning it in a comment saying that this is a semantics-preserving optimization, that duplicates what is done in `dcalc/optimizations.ml`